magento/adobe-stock-integration
# [WIP]#1523: Switching between Views does not change the selected folder. [Media Gallery]

### Description (*)

This PR will implement the fix to change the selected folder when switching views in Media Gallery.
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration https://github.com/magento/adobe-stock-integration/issues/1523: Switching between Views does not change the selected folder. [Media Gallery]

### Manual testing scenarios (*)


1. Go **Content - Pages** click **Add New Page**
2. Expand **Content** click **Show / Hide Editor**, **Insert Image**...
3. Create a main folder named _folder_ for example and select it
4. Click on **Default View** dropdown
5. Click **Save View As**..., and click **Save all changes** ( ➡️ )
 _now you have a New View with the selected folder_
6. Close the Media Gallery and go to Catalog - Categories
7. Expand Content and click Select from Gallery
_folder is selected_
8. Try to switch between **New View** and **Default View**
9. The folder selected for that view must be highlighted/selected, 
i.e.: If **New View** is selected, _folder_ should be selected, 
**Catalog/category** folder is selected if the **Default view** is selected
